src/Bitcoin/CompactFilter.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
```

Comment:
not sure if it's worth to put `NamedFieldPuns` behind the CPP conditional?

Reply:
IMO it's good to minimize the amount of stuff behind macros.  It ends up being hard to maintain.  Let's leave it as is :)